README.md

```diff
@@ -1,113 +1,29 @@
-
-cd yolov5_ros/yolov5
-
-sudo pip install -r requirements.txt
-```
-
-## Basic Usage
-
-1. First, make sure to put your weights in the [weights](https://github.com/qq44642754a/Yolov5_ros/tree/master/yolov5_ros/yolov5_ros/weights) folder. 
-2.  The default settings (using `yolov5s.pt`) in the `launch/yolo_v5.launch` file should work, all you should have to do is change the image topic you would like to subscribe to:
-
-```
-roslaunch yolov5_ros yolo_v5.launch
-```
-
-  
-  Alternatively you can modify the parameters in the [launch file](https://github.com/qq44642754a/Yolov5_ros/tree/master/yolov5_ros/launch/yolo_v5.launch), recompile and launch it that way so that no arguments need to be passed at runtime.
-
-### Node parameters
-
-* **`image_topic`** 
-
-    Subscribed camera topic.
-
-* **`weights_path`** 
-
-    Path to weights file.
-
-* **`pub_topic`** 
-
-    Published topic with the detected bounding boxes.
-    
-* **`confidence`** 
-
-    Confidence threshold for detected objects.
-    
-
-
+# objectdetect_yolov5ros_pkg
+## Description
+This repository is a fork of the original [Yolov5_ros](https://github.com/qq44642754a/Yolov5_ros) package developed by Zhitao Zheng. We extend our heartfelt thanks to Zhitao Zheng for their foundational work. This project has been modified and tailored specifically for IcebergASV, enhancing its capabilities and adaptability. Significant alterations include customization for seamless integration with IcebergASV's systems and the addition of functionalities to run both with a camera setup and in a simulation environment. 
+## General Requirements
+- **YOLOv5 Installation**: Follow the documentation on the [YOLOv5 Installation Guide](https://app.gitbook.com/o/vtYvioW5qkBb75Erv7gv/s/PzWCobYwRWwuEeL79eAC/yolov5-setup/installing-yolov5) in GitBook. Since a virtual environment is created in the documentation **make sure to activate it in every terminal you use** for this package. 
+- **YOLOV5_PATH Enviornment Variable Creation**: Follow the documentation on the [YOLOV5_PATH Enviornment Creation](https://app.gitbook.com/o/vtYvioW5qkBb75Erv7gv/s/PzWCobYwRWwuEeL79eAC/yolov5-setup/yolov5_path-environment-variable) in GitBook.
+## Run with Intel RealSense D435 Camera
+### Requirements
+- **Intel RealSense D435 Camera Setup**: Follow the [Sensorland](https://github.com/IcebergASV/Sensorland) README.md file to complete the installation. 
+### Usage
+1. Open a terminal and launch the ```roslaunch realsense2_camera rs_camera.launch```. The object_detection_pkg subscribes to the /camera/color/image_raw publisher for live camera feed used for object detection.
+2. Open a new terminal and launch the ```roslaunch yolov5_ros yolo_v5.launch ```
+
+## Run in Gazebo Simulation
+### Requirements
+- **Gazebo**
+### Usage
+1. Open a terminal and launch the ```roslaunch topic```. The object_detection_pkg subscribes to the /webcam/image_raw publisher for live camera feed in Gazebo Simulation used for object detection.
```

Comment:
Accidentally made a comment from Harnoor's account. But do you mean launch gazebo instead of "roslaunch topic"?

Reply:
Yes, I just left it like that! I wasn't sure what the command was for that